### DISCUSSION TITLE
An option to class airfields that only have an apron as empty

### DISCUSSION THREAD

**Discussion by bcol80** [albar965/littlenavmap#1009] [Ideas]
There's an option to disable the view of 'empty' airfields on the map and the manual states that 'empty' airfields are "An airport that has no taxiways, no parking positions or gates, no aprons and is not an add-on airport and is not a water airport." but in MSFS there is an absolte ton of default airfields that lack parking positions and taxiways but often have a tiny little bit of 'apron' to mask out something like a parked plane in the aerial imagery. These are for all intents and purposes empty airfields but because of the tiny bit of apron they do not get hidden when you hide empty airfields.  So could there be an option to disregard the 'apron' so these empty airfields can be hidden on the map? Cheers

Here's a few random examples of MSFS airfields have are basically empty aside from a tiny bit of apron:
![image](https://user-images.githubusercontent.com/87935689/224015807-7ae37e5c-91d5-4f6e-9c3d-3895aace4e6a.png)
![image](https://user-images.githubusercontent.com/87935689/224015870-43d48c79-a171-49b9-b6ab-a6addc3ee00f.png)
![image](https://user-images.githubusercontent.com/87935689/224015923-7a3679a0-f801-4d93-ba50-a6193937c630.png)
![image](https://user-images.githubusercontent.com/87935689/224015980-de2c3912-09ba-466f-a794-c3805feddebe.png)
![image](https://user-images.githubusercontent.com/87935689/224016129-179ed3a1-3e5f-48fa-ab47-672849e30eb9.png)

**Comment by albar965** [comment]
Makes sense and is doable. I can add this change to the next update.

**Reply by albar965** [reply]
Made an issue for this: #1010
Thanks for the screenshots and examples!
